examples/saml-sso/keycloak_saml.md

```diff
@@ -35,7 +35,7 @@ If you imported the JSON file, you should have a "ror" realm, and a SAML client
 
 ### Configure Keycloak to work with ROR
 
-First we want to create a new dedicated "ror" realm, so we don't interfere with any other use of this Keycloak installation.
+First, we want to create a new dedicated "ror" realm, so we don't interfere with any other use of this Keycloak installation.
```

Comment:
Yes! The attention to details makes or breaks the good first impression about how much care we put in the product! ✅